.travis.yml

```diff
@@ -28,6 +28,9 @@ before_install:
   - ulimit -a
   - mkdir builds
   - pushd builds
+  # Build into own virtualenv
```

Comment:
Do we always want to do this? Just curious. A bit more explanation in the comment would help.